src/xdsl/dialects/std.py

```diff
@@ -41,3 +43,25 @@ class Return(Operation):
     @staticmethod
     def get(*ops: Union[Operation, SSAValue]) -> Return:
         return Return.build(operands=[[op for op in ops]])
+
+
+@irdl_op_definition
+class Branch(Operation):
+    name: str = "std.br"
+    arguments = VarOperandDef(AnyAttr())
+
+    @staticmethod
+    def get(*ops: Union[Operation, SSAValue], successor=Block) -> Branch:
+        return Branch.build(operands=[[op for op in ops]], successor=[Block])
+
+
+@irdl_op_definition
+class ConditionalBranch(Operation):
+    name: str = "std.cond_br"
+    arguments = VarOperandDef(AnyAttr())
+
+    @staticmethod
+    def get(*ops: Union[Operation, SSAValue],
```

Comment:
I think the best here would be to have `get(succ_true, [succ_true_args], succ_false, [succ_false_args])

Reply:
Changed.